README.md

```diff
@@ -1,5 +1,3 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
 ## Available Scripts
 
 In the project directory, you can run:
```

Comment:
You have done well to add a description, there are few things missing though,  please see bellow suggestions
 It is recommended to guide the user step by step on how to run the app
```suggestion

1. move to the project directory within your terminal `cd  ~/Projecte_directory`
2. install dependencies ` npm run install`
3. start the app `npm run start `
```

Please be mindful of styling, feel free to google the internet for ideas.

Reply:
1. move to the project directory within your terminal `cd  ~/Projecte_directory`
2. install dependencies ` npm run install`
3. start the app `npm run start `